win32ss/user/ntuser/win32.h

```diff
@@ -52,120 +50,6 @@ extern HANDLE hModuleWin;    // This Win32k Instance.
 extern struct _CLS *SystemClassList;
 extern BOOL RegisteredSysClasses;
 
-#include <pshpack1.h>
-// FIXME: Move to ntuser.h
```

Comment:
2. There were FIXME that argues to move the structures.

Reply:
3. When I moved these, compilation passed.

Reply:
This comment was erroneous. These structures are only valid in kernel mode (within win32k).

Reply:
OK.